geomstats/geometry/discrete_curves.py

```diff
@@ -700,15 +700,15 @@ def __init__(self, space):
 
     @staticmethod
     def riemann_sum(func):
-        """Compute the left Riemann sum approximation of the integral.
+        r"""Compute the left Riemann sum approximation of the integral.
 
         Compute the left Riemann sum approximation of the integral of a
         function func defined on the unit interval, given by sample points
         at regularly spaced times
 
         .. math::
-            t_i = i / (k_landmarks),
-            i = 0, ..., k_landmarks - 1
+            t_i = i / (k\_landmarks), \\
```

Comment:
I think is better to do one of the following:

1. `k` instead of `k_landmarks`

2. if 1. not enough for understanding do `k`, and then write "where :math:`k` is the number of landmarks"

3. `k_{landmarks}`

I have no preference.